test/e2e/suite/command/discover.go

```diff
@@ -48,6 +48,7 @@ func discoverKeyWords(verbose bool, descs ...ocispec.Descriptor) []string {
 }
 
 var _ = Describe("ORAS beginners:", func() {
+	const DeprecationMessageTableFormat = "Format \"table\" is deprecated and will be removed in a future release.\n"
```

Comment:
nit: no capital letter for local variable names?

Reply:
We can consider putting it in `e2e/internal/testdata/feature/feature.go`

Reply:
Moved the variable to `e2e/internal/testdata/feature/feature.go`